### ISSUE TITLE
integrate Map with mocked services

### ISSUE THREAD

**Issue by blanche** [pintheworld/pintheworld#37]
Assuming we have two methods:
- getCity(), which returns a city object like {name, lat, long}
- submitGuess(cityName, userLat, userLong) which just prints its parameter to the console

integrate the map so that it covers the following process
1. calls getCity() to get the city the user has to guess 
2. display map and city name
3. let the user make a guess by clicking on the map
4. show the users guess and the real city coordinates on the map
5. call submitGuess(cityName, userLat, userLong) (this method can just print the parameters to the console)
6. clear the map and start at 1) 

for now this will continue with the same city vor every round, but the getCity() method will return random cities when the services are implemented

**Comment by blanche** [comment]
example getCity() function:

```
function getCity(){
    return [{
        "lat": 51.509864999999998,
        "name": "London",
        "long": -0.118092,
        "id": 5629499534213120
    }];
}
```

**Comment by stellaxxiang** [comment]
solved by #31
